src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj

```diff
@@ -125,6 +126,21 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <Target Name="ResolveReadyToRunCompilers" DependsOnTargets="ResolveRuntimeFilesFromLocalBuild">
+    <!-- The following property group can be simplified once runtime repo switches over to SDK 6.0 drop -->
```

Comment:
Looks like these properties are a duplicate of the ones in the touched file above. Would it make sense to extract them into a props file and import them in both projects to avoid the duplication?

Reply:
I agree that's a straightforward copy but I suspect it can't be props as it's a target with a state. I think we could make it an Import but frankly speaking 90% of the code will be deleted after Crossgen1 and SDK 5.0 are retired so I'm not sure if it's worth the effort, ultimately the remaining Crossgen2 variant should boil down to less than half dozen lines of msbuild script code.

Reply:
If this is just temporary then I'm fine with the duplication

Reply:
Removed duplication with https://github.com/dotnet/runtime/pull/48201.